scapy/config.py

```diff
@@ -942,12 +946,14 @@ def crypto_validator(func):
     This a decorator to be used for any method relying on the cryptography library.  # noqa: E501
     Its behaviour depends on the 'crypto_valid' attribute of the global 'conf'.
     """
+
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok